core/src/Library.js

```diff
@@ -69,10 +69,20 @@ const Router = typeof document !== 'undefined'
   : StaticRouter
 
 export class Library extends React.Component {
+  static propTypes = {
+    baseUrl: PropTypes.string
+  }
+
+  static defaultProps = {
+    baseUrl: '/'
```

Comment:
I think we would want the default to be `''` but also `undefined` *should* work for react router

Reply:
Yeah, I think we can nuke the prop type and default prop since we're now forwarding to react router.